### PR TITLE
Webui bz report offline

### PR DESCRIPTION
**Pull request by rvykydal**
Addresses https://issues.redhat.com/browse/INSTALLER-3637
TODO:
- [ ] tests (can be tricky given we are testing becoming offline via remote tests?)
- [x] need to fix the icon, wasn't able to import the HiStatusOffline from https://react-icons.github.io/react-icons/search?q=offline which I think is in the mockup

Video:
https://rvykydal.fedorapeople.org/webui/error_handling/webui-bz-report-offline.webm

To check locally you can apply updates image in your live installation of some reasonably recent image in terminal by running:
```
sudo /usr/libexec/anaconda/apply-updates https://rvykydal.fedorapeople.org/webui/error_handling/updates.offline.im
```

### CODE REVIEW

File: ui/webui/src/components/Error.jsx [hunk 1]
```diff
@@ -127,7 +129,9 @@ export const BZReportModal = ({
                     />
                     <FormHelperText isHidden={false}>
                         <HelperText>
-                            <HelperTextItem>{_("Reporting an issue will send information over the network. Plese review and edit the attached log to remove any sensitive information.")}</HelperTextItem>
+                            {isConnected
+                                ? <HelperTextItem> {_("Reporting an issue will send information over the network. Plese review and edit the attached log to remove any sensitive information.")} </HelperTextItem>
```
Comment (KKoukiou) [inline ui/webui/src/components/Error.jsx:133]: s/Plese/Please

Comment (KKoukiou) [reply ui/webui/src/components/Error.jsx:133]: oops this exists upstream already.

**Review by KKoukiou — COMMENTED** [review]
Looks great. you 're right about the testing that it might be hard.

**Review by KKoukiou — COMMENTED** [review]
Regarding the icon you mentioned from https://react-icons.github.io/react-icons/search?q=offline this is a whole new librbary and not Patternfly. So I think we should stick to Patternfly available icons. 
If some icons is missing there, report that upstream https://github.com/patternfly/patternfly-react/issues/

File: ui/webui/src/components/Error.jsx (hunk 1 above)
Comment (rvykydal) [reply ui/webui/src/components/Error.jsx]: Fixed

**Comment by rvykydal** [comment]
> Regarding the icon you mentioned from https://react-icons.github.io/react-icons/search?q=offline this is a whole new librbary and not Patternfly. So I think we should stick to Patternfly available icons. If some icons is missing there, report that upstream https://github.com/patternfly/patternfly-react/issues/

@garrett any tip? I am failing to find something suitable available.

**Comment by rvykydal** [comment]
I would suggest to add the tests in a separate issue.

**Comment by AdamWill** [comment]
Fedora bug report for Beta FE purposes: https://bugzilla.redhat.com/show_bug.cgi?id=2234964

File: ui/webui/src/components/Error.jsx [hunk 2]
```diff
@@ -97,14 +98,16 @@ export const BZReportModal = ({
               <Stack hasGutter>
                   <FormHelperText isHidden={false}>
                       <HelperText>
-                          <HelperTextItem>{_("Reporting an issue will send information over the network. Plese review and edit the attached log to remove any sensitive information.")}</HelperTextItem>
+                          {isConnected
+                              ? <HelperTextItem> {_("Reporting an issue will send information over the network. Please review and edit the attached log to remove any sensitive information.")} </HelperTextItem>
+                              : <HelperTextItem icon={<DisconnectedIcon />}> {_("Network not available. Configure the network in the top bar menu to report the issue.")} </HelperTextItem>}
```
Comment (M4rtinK) [inline ui/webui/src/components/Error.jsx:103]: I think this is good for now in the Fedora Workstation environment, but we should take a note to handle this somehow better for other environments. Eq. boot.iso currently does not have a top bar & other Live spins could have different mechanisms for this.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Pull request merged**